mods/seasonal/random-teams.js

```diff
@@ -37,6 +37,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: "Buzzing of the Storm",
 				evs: {def:4, atk:252, spe:252}, nature: 'Jolly',
 			},
+			'Biggie': {
+				species: 'Snorlax', ability: 'Fur Coat', item: 'Leftovers', gender: 'M', //ask gender
+				moves: ['drainpunch', ['diamondstorm', 'kingsshield'][this.random(2)], ['knockoff', 'precipiceblades'][this.random(2)]], // ask about moveset
```

Comment:
confirm moveset before this gets merged, generally hes just going to have to pair up moves into sets of possible choices like now or limit it to 3.